code/obj/decal.dm

```diff
@@ -809,3 +809,65 @@ obj/decal/fakeobjects/teleport_pad
 /obj/decal/tile_edge/floorguide/arrow_s
 	name = "Directional Navigation Guide"
 	icon_state = "endpiece_s"
+
+/obj/decal/slipup
+	name = ""
+	desc = ""
+	anchored = ANCHORED
+	mouse_opacity = 0
+	icon = null
+	icon_state = null
+	alpha = 0
+	opacity = 0
+	pixel_x = 0
+	pixel_y = 8
+	plane = PLANE_NOSHADOW_ABOVE
+
+	New(var/location = null, var/state = null, var/mob/target = null)
+		if (!istype(src, /obj/decal/slipup/clumsy))
```

Comment:
??? just clean up and fix the code instead of this weird junk

Reply:
I'm not sure what kind of shitcode demon possessed me there. Sorry about that